lib/commands/context.js

```diff
@@ -329,6 +332,18 @@ async function getChromedriverPort (portSpec) {
   return foundPort;
 }
 
+helpers.isChromedriverAutodownloadEnabled = function isChromedriverAutodownloadEnabled () {
+  try {
+    this.ensureFeatureEnabled(CHROMEDRIVER_AUTODOWNLOAD_FEATURE);
```

Comment:
you can use `isFeatureEnabled` if you just want the boolean value instead of using try/catch...